app/_hub/kong-inc/ai-rag-injector/how-to/_index.md

```diff
@@ -62,7 +65,20 @@ plugins:
 
 ### 2. Make a AI request to the AI Proxy Advanced plugin
 
-The AI RAG Injector plugin uses the OpenAI `text-embedding-3-large` model to generate embeddings for the content and stores them in Redis. 
+User will need to split the document into chunks by their own, typically use tools like`langchain_text_splitters`, and feed the splitted chunk using Admin API. 
+	
+The following example shows how to ingest content to the vector database for building the knowledge base. The AI RAG Injector plugin uses the OpenAI `text-embedding-3-large` model to generate embeddings for the content and stores them in Redis.
+
+```bash
+curl localhost:8001/ai-rag-injector/3194f12e-60c9-4cb6-9cbc-c8fd7a00cff1/ingest_chunk \
```

Comment:
Which plugin's ID is this? The related AI Proxy/Proxy Advanced plugin?

Reply:
Nope, it's `ai-rag-injector` ID.